core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -210,7 +215,10 @@ object ConsumerPerformance {
    
     val useNewConsumer = options.has(useNewConsumerOpt)
     
-    val props = new Properties
+    val props = if (options.has(consumerConfigOpt))
+      Utils.loadProps(options.valueOf(consumerConfigOpt))
```

Comment:
Interesting. Ideally, specified command-line settings would override the properties, but it seems like that would be a bigger change. Perhaps this should behave as the `ConsoleConsumer` until we can overhaul this. It looks like the change here does that, right?